### PR TITLE
Add UARTService; add standard library files

### PR DESCRIPTION
**Pull request by dhalbert**
- UARTService now working with https://github.com/adafruit/Adafruit_CircuitPython_BluefruitConnect packet handling.
- Various boilerplate files were missing, including .travis.yml. PyPi not included for now, since we have no wrapper libraries on Linux for `bleio` functionality.

### CODE REVIEW

File: adafruit_ble/uart.py [hunk 1]
```diff
@@ -20,53 +20,104 @@
...
+from bleio import UUID, Characteristic, Service, Peripheral, CharacteristicBuffer
 
-import bleio
-
-from .uuid import UUID
-from .characteristic import Characteristic
-from .service import Service
-from .peripheral_server import PeripheralServer
 
 class UARTService:
```
Comment (tannewt) [inline adafruit_ble/uart.py]: Should this be UARTPeripheral or something else instead of service because it cannot be used with other services?

Comment (dhalbert) [reply adafruit_ble/uart.py]: I agree `UARTService` is probably misleading. How about `UARTServer`? It doesn't really act as either a subclass of `Service` or a subclass of `Peripheral`.

File: README.rst [hunk 2]
```diff
@@ -30,8 +30,13 @@ Usage Example
...
 .. code-block:: python
 
-    import ble
-    # Put an advertising or simple peripheral example here.
+    from adafruit_ble.uart import UARTService
+
+    uart_service = UARTService()
+    uart_service.start_advertising()
+
+    uart_service.write('abc')
```
[the remark this answers is not in the document]
Comment (dhalbert) [reply README.rst]: Thanks! Actually there should be a loop waiting for `uart_service.connected`. I'll add that.

File: adafruit_ble/uart.py (hunk 1 above)
Comment (dhalbert) [reply adafruit_ble/uart.py]: And it really is officialy a server, in terms of it being a GATT server.

**Pull request merged**